src/Service.Host/client/src/components/requisitions/reducers/requisitonReducer.js

```diff
@@ -43,43 +43,47 @@ function reducer(state, action) {
                     document2: action.payload.newValue
                 };
             } else if (action.payload.fieldName === 'storesFunction') {
-                let newState = { ...state, storesFunction: action.payload.newValue };
+				
+                const mapping = { M: 'Y', A: 'N', X: null };
+                let newState = {
+                    ...state,
+                    storesFunction: action.payload.newValue,
+                    manualPick: mapping[action.payload.newValue?.manualPickRequired]
```

Comment:
the question mark after newValue presumably means we might anticapte this key to be null. null as an object key is kind of a strange  concept - I'd probably verify newValue isn't null before trying to use it as a key to access values of the 'mapping' object